pitaya-sharp/NPitaya/src/PitayaCluster.RPC.cs

```diff
@@ -121,6 +123,8 @@ static void DispatchRpc(RpcClient rpcClient, IntPtr rpc, Protos.Request req)
                 }
                 finally
                 {
+                    start.Stop();
+                    _metricsReporter?.ObserveHistogram(RPC_LATENCY_METRIC, (float)start.Elapsed.TotalSeconds, null);
```

Comment:
Won't we lose precision with this one? Maybe it's better to use `TotalMilliseconds`.

Reply:
No, `TotalSeconds` is a double, so we shouldn't have issues here, there is enough precision.